enhancements/single-node/aos-sno-pair.md

```diff
@@ -0,0 +1,143 @@
+---
+title: aos-sno-pair
+authors:
+- @mshitrit
+  reviewers:
+- TBD
+  approvers:
+- TBD
+
+creation-date: 2021-07-12
+last-updated: 2021-07-12
+status: implementable
+---
+
+# Atomic OpenShift Single-Node-Pair
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+This enhancement describes setting up a two nodes cluster with a reduced footprint which will serve edge cases.
+
+## Motivation
+
+Some companies have a need for a highly available container management solution that fits within a reduced footprint.
+- The hardware savings are significant for customers deploying many remote sites (kiosks, branch offices, restaurant chains, etc), most notably for edge computing and RAN specifically.
+- The physical constraints of some deployments prevents more than two nodes (planes, submarines, satellites, and also RAN).
+- Some locations will not have reliable network connections or limited bandwidth (once again submarines, satellites, and disaster areas such as after hurricanes or floods)
+  
+### Goals
+
+Provide the ability for all-in-one OpenShift clusters to operate as a **predefined** pair in an **active-active** configuration, detect when its peer has died and **automatically** take over its workloads after ensuring it is safe to do so.
+
+### Non-Goals
+
+- Management as a single entity (pod/namespace replication).
+- **Replication of application data** between appliances.
+- DR for services in clusters with **more than a single node.**
+- DR for services running on **two or more all-in-one clusters.**
+- Enable services uptime during a **remote cluster upgrade process.**
+
+## Proposal
+
+Assuming workloads that require some form of coordination between the two nodes (either at-most-one semantics or simply to reduce machine load during normal operation):
+
+- Pacemaker and Corosync from the RHEL-HA cluster stack are deployed [in Pods](https://github.com/beekhof/installer/blob/aio/pod-cluster.yaml.in) on both OCP clusters, and they are configured to form a two node cluster.
```

Comment:
That's a 2 node pacemaker cluster, not a 2 node OpenShift cluster, right?

Reply:
That is two Single Node OpenShift (SNO) clusters, each SNO is independent and unaware of the other one.
Pacemaker is involved in  maintaining the High Availability stack that is on top.

Reply:
Saying that two single node instances of OpenShift form a two node OpenShift cluster is confusing. The nodes are not part of the same OpenShift cluster.

Reply:
Agreed.  Lets standardize on either `RHEL-HA cluster` or `Pacemaker cluster` to refer to the higher level 2-node entity